test/testintersections.c

```diff
@@ -231,21 +231,21 @@ static void loop(void *arg)
                     num_lines = 0;
                 } else {
                     add_line(
-                        (float)(SDL_rand() % 640),
-                        (float)(SDL_rand() % 480),
-                        (float)(SDL_rand() % 640),
-                        (float)(SDL_rand() % 480));
+                        (float)SDL_rand_n(640),
+                        (float)SDL_rand_n(480),
+                        (float)SDL_rand_n(640),
+                        (float)SDL_rand_n(480));
```

Comment:
Would this make sense?
```suggestion
                       640.f * SDL_rand_float(),
                       480.f * SDL_rand_float(),
                       640.f * SDL_rand_float(),
                       480.f * SDL_rand_float());
```

Reply:
The rand_n version will always be at whole pixels, the float version could be located between pixels. I deliberately didn't touch this kind of stuff because I don't understand the intentions of the code.

Reply:
That's understandable. Ignore my comment then.